mod/dfrn_request.php

```diff
@@ -433,10 +433,10 @@ function dfrn_request_post(App $a)
 			}
 
 			// "Homecoming" - send the requestor back to their site to record the introduction.
-			$dfrn_url = bin2hex(System::baseUrl() . '/profile/' . $nickname);
+			$dfrn_url = bin2hex('profile/' . $nickname);
 			$aes_allow = ((function_exists('openssl_encrypt')) ? 1 : 0);
 
-			$a->redirect($parms['dfrn-request'] . "?dfrn_url=$dfrn_url"
+			$a->internalRedirect($parms['dfrn-request'] . "?dfrn_url=$dfrn_url"
```

Comment:
Isn't `dfrn-request` an absolute URL?

Reply:
I was searching for some information about this param and thought it was a relative URL. But I trust your experience much more than my research ;-) ..